src/chapters/data-manipulation/data-manipulation.rst

```diff
@@ -1,2 +1,162 @@
 Data Manipulation
-=================
\ No newline at end of file
+=================
+
+.. _readingDataManipulation:
+
+Readings
+--------
+
+The readings and exercises are a little different for this chapter.  Both have been combined into 2 
```

Comment:
```suggestion
You will find the assigned reading within this lesson's jupyter notebooks.  Please fork and clone the notebooks to start learning about data manipulation.
```

Reply:
I would like to keep forking to the exercises page.